Core/src/org/sleuthkit/autopsy/healthmonitor/HealthMonitorDashboard.java

```diff
@@ -58,8 +58,9 @@
     
     private final static Logger logger = Logger.getLogger(HealthMonitorDashboard.class.getName());
     
-    private final static String ADMIN_ACCESS_FILE_NAME = "adminAccess"; // NON-NLS
-    private final static String ADMIN_ACCESS_FILE_PATH = Paths.get(Places.getUserDirectory().getAbsolutePath(), ADMIN_ACCESS_FILE_NAME).toString();
+    private final static String ADMIN_ACCESS_FILE_NAME = "_aiaa"; // NON-NLS
+    private final static String ADMIN_ACCESS_FILE_PATH = Paths.get(Places.getUserDirectory().getAbsolutePath(),
+            "config" + File.separator + ADMIN_ACCESS_FILE_NAME).toString();
```

Comment:
Paths.get takes care of returning a Path with the path component separators between each item in the input list.  The concatenation with File.separator is not needed here.